### PR TITLE
Delete telemetry fix

### PR DESCRIPTION
**Pull request by korshaknn**
closes [1520](https://app.zenhub.com/workspaces/vision-sdk-5ca52e6c027e3e7234ac7512/issues/mapbox/mapbox-vision/1520)

### CODE REVIEW

File: MapboxVision/src/main/java/com/mapbox/vision/sync/syncmanager/TelemetrySyncManager.kt [hunk 1]
```diff
@@ -133,11 +134,17 @@ class TelemetrySyncManager(
...
         if (totalTelemetrySize > MAX_TELEMETRY_SIZE) {
             var bytesToRemove = totalTelemetrySize - MAX_TELEMETRY_SIZE
-            val sortedTelemetryDirs = rootTelemetryDirectory
-                .listFiles()
-                .sortedBy { it.name }
+            val telemetryDirs = mutableListOf<File>()
 
-            for (telemetryDir in sortedTelemetryDirs) {
+            val countryDirs = rootTelemetryDirectory.listFiles()
+            countryDirs?.forEach { dir ->
```
Comment (alexander-kulikovskii) [inline MapboxVision/src/main/java/com/mapbox/vision/sync/syncmanager/TelemetrySyncManager.kt]: maybe change _dir_ to _countryDir_, _it_ -> _timestampDir_

Comment (korshaknn) [reply MapboxVision/src/main/java/com/mapbox/vision/sync/syncmanager/TelemetrySyncManager.kt]: agree, done

File: MapboxVision/src/main/java/com/mapbox/vision/sync/syncmanager/TelemetrySyncManager.kt [hunk 2]
```diff
@@ -81,7 +81,8 @@ class TelemetrySyncManager(
 
         val dirFile = File(path)
 
-        removeTelemetryOverQuota(dirFile.parentFile)
+        // up to Telemetry dir
```
Comment (yunikkk) [inline MapboxVision/src/main/java/com/mapbox/vision/sync/syncmanager/TelemetrySyncManager.kt]: Could you pls make this comment clearer?

Comment (korshaknn) [reply MapboxVision/src/main/java/com/mapbox/vision/sync/syncmanager/TelemetrySyncManager.kt]: @yunikkk done

**Pull request merged**